app/Domain/Ideas/Templates/advancedBoards.blade.php

```diff
@@ -24,48 +24,39 @@
             <h5><?php $tpl->e(session('currentProjectClient') . ' // ' . session('currentProjectName')); ?></h5>
             <h5>{{ session('currentProjectClient') . ' // ' . session('currentProjectName') }}</h5>
             @if (count($allCanvas) > 0)
-                <x-global::content.context-menu label-text="<i class='fa-solid fa-ellipsis-v'></i>" contentRole="link"
-                    position="bottom" align="start" class="headerEditDropdown btn btn-transparent">
-
-                    <x-slot:menu>
+                <span class="dropdown dropdownWrapper headerEditDropdown">
```

Comment:
Why was this reverted? I assume this is the 3 dot menu on the top right hand side? That should use the dropdown component.

Reply:
i was having issues in this fiel so I reverted it, and started replacing it piece by piece again